src/dropdown.ts

```diff
@@ -726,7 +707,7 @@ export default class GlideCoreDropdown
                         data-id=${id}
                         label=${ifDefined(label)}
                         removable
-                        size=${this.size}
+                        size="large"
```

Comment:
Design also asked me to remove `size` from Tag and have all tags be large. I'll be doing that next. I set it to large now so I could include both changes to Dropdown in a single changeset, rather than having a Dropdown changing in the upcoming changeset for Tag.